stubs/common/Model.stub

```diff
@@ -66,6 +66,13 @@ abstract class Model implements \JsonSerializable, \ArrayAccess
      * @return \Illuminate\Database\Eloquent\Builder<static>
      */
     public static function with($relations);
+
+    /**
+     * Begin querying the model.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder<static>
+     */
+    public static function query();
```

Comment:
This is fine, however, I'm 90% sure that this stub is not being used given that there's an extension that calculates what the return type is

Reply:
Without this stub method - tests in 9,10 version laravel failed

Reply:
That's because in your test you have:

```php
assertType('Illuminate\Database\Eloquent\Builder<App\User>', (new $className())->query());
```

which is calling a **static** function non-statically (and thus the static extension is not being used). This should be:

```php
assertType('Illuminate\Database\Eloquent\Builder<App\User>', (new $className())->newQuery());
```